src/js/components/TaskListComponent.jsx

```diff
@@ -69,10 +88,22 @@ var TaskListComponent = React.createClass({
             key={task.id}
             onToggle={props.onTaskToggle}
             task={task}
-            taskHealthMessage={props.getTaskHealthMessage(task.id)}/>
+            taskHealthMessage={props.getTaskHealthMessage(task.id)} />
         );
       })
       .value();
+
+    if (targetInstances > props.tasks.length && nodes.length < itemsPerPage) {
+      let numPlaceholders = Math.min(itemsPerPage - nodes.length,
+        targetInstances - nodes.length);
+
+      for (let i = 0; i < numPlaceholders; i++) {
+        nodes.push(<PlaceholderTaskListItemComponent key={i}/>);
+      }
+    }
+    nodes.push(<PlaceholderTaskListItemComponent key={123}/>);
```

Comment:
This is probably a remain from developing the style. it should be removed otherwise there is always one empty entry in the list.